.buildkite/pipeline.release.stable.yml

```diff
@@ -0,0 +1,37 @@
+- name: ":buildkite: :package: Publish Docker image to Buildkite Packages"
```

Comment:
We push the stable version to Buildkite Packages as well, so we can use it internally. And it will make it easier to switch between nightly build and stable version internally. We only need to change the version number.